integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/ad-hoc/SetSpecificAttributeValue.feature

```diff
@@ -0,0 +1,37 @@
+# SPDX-FileCopyrightText: Contributors to the GXF project
+#
+# SPDX-License-Identifier: Apache-2.0
+
+@SmartMetering @Platform @SmartMeteringAdHoc
+Feature: SmartMetering AdHoc
+  As a grid operator
+  I want to be able to set a specific attribute value in a device
+  So I can change the configuration of a device
+
+  Scenario Outline: Set watchdog timer for a <protocol> <version> device
+    Given a dlms device
+      | DeviceIdentification | <deviceIdentification> |
+      | DeviceType           | SMART_METER_E          |
+      | Protocol             | <protocol>             |
+      | ProtocolVersion      | <version>              |
+    When the Set Specific Attribute value request is received
```

Comment:
Very minor comment: value is part of the request name so I expect it to be with capital V: _Set Specific Attribute Value_